README.md

```diff
@@ -62,6 +62,16 @@ coder.dump([1]) # => "\x01[1]"
 coder.load("\x00---\n:foo: 42") # => { foo: 42 }
 ```
 
+### `SingleBytePrefixVersionWithStringBypass`
+
+Works like `Paquito::SingleBytePrefixVersion` except that versions `253`, `254` and `255` are reserved for serializing string
+in an optimized way.
+
+When the object to serialize is an `UTF-8`, `ASCII` or `BINARY` string, rather than to invoke the underlying serializer, it simply
+prepend a single byte to the string which indicate the encoding.
```

Comment:
```suggestion
When the object to serialize is an `UTF-8`, `ASCII` or `BINARY` string, rather than invoking the underlying serializer, it simply
prepends a single byte to the string which indicates the encoding.
```